Refine docs — introduction (#1305)
software-mansion/protostar: website/docs/tutorials/01-introduction.md

# Introduction

Protostar is a toolchain for developing StarkNet smart contracts that helps with tasks such as [dependencies management](06-dependencies-management.md), [project compilation](05-compiling.md), and [testing contracts](07-testing/README.md).
It is inspired by [Foundry](https://github.com/foundry-rs/foundry).

## Target audience

> This guide assumes that you have basic knowledge of [Cairo, StarkNet](https://www.cairo-lang.org/docs/), and Git.

Reviewer: and was better

Author: IMO it was worse.
- There was a grammar error (it should be "these guides", or "this guide" not "this guides").
- "And" was used twice in a list of three items.

Reviewer: Ah you are right, git is there, then lets change it

## Learning objectives

> After reading this guide, you will be able to:

Reviewer: ```suggestion
After reading this guides, you will know how to:
```

Author: [use "guides"](https://github.com/software-mansion/protostar/pull/1305/commits/55cb26ede3e3b1cb8a0e7bae76e23993d65375dc)
